src/live.asp.net/Services/FeedService.cs

```diff
@@ -78,86 +78,73 @@ public AtomFeed()
         [XmlNamespaceDeclarations]
         public XmlSerializerNamespaces Namespaces { get; set; }
 
-        [XmlElement("id")]
-        public string Id { get; set; }
+        public string id { get; set; }
```

Comment:
Okay. Thanks. I read the document and will change accordingly. Sorry for not paying attention to it early on.